setup/system.sh

```diff
@@ -164,6 +185,16 @@ if [ -z "$DISABLE_FIREWALL" ]; then
 	# Install `ufw` which provides a simple firewall configuration.
 	apt_install ufw
 
+	# Make sure the system has a default policy to accept incoming connections
+	sed -i "s/DEFAULT_INPUT_POLICY.*/DEFAULT_INPUT_POLICY=\"ACCEPT\"/" /etc/default/ufw
+
+	# If IPV6 is disabled we should disable it in the ufw defaults
+	if [ -z "$PUBLIC_IPV6" ]; then
+		sed -i "s/IPV6.*/IPV6=no/" /etc/default/ufw
```

Comment:
I'm not sure about this. ufw should be smart enough to not use IPv6 if it's not in the system.